### PR TITLE
Project review task and code review

### PR DESCRIPTION
**Pull request by edi-sipka**
This task is about code review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        </a>
+        <img class="logo" alt="youtube logo" src="Images/youtube-logo.png" />
+      </div>
+
+      <div class="search-box">
+        <input type="text" class="search-input" placeholder="Search..." />
+        <div class="search-bar">
+          <img class="search-button" alt="search bar" src="images/search.png" />
+        </div>
+      </div>
```
Comment (edi-sipka) [inline index.html:35]: Please use nav instead of div by HTML5 semantic rules.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            width="860"
+            height="485"
+            src="https://www.youtube.com/embed/lbEYptkDm0A"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          ></iframe>
+          <div class="title">
+            <h2>Faithless live on Exit Festival 2015 [ Minimal Techno Mix ]</h2>
```
Comment (edi-sipka) [inline index.html:61]: I am kindly suggesting you to use h1 instead of h2, because, it is the main heading.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+              Why he would to it? <br />Nimes is active for a sound.
+            </p>
+
+            <p class="sub-text-user">Mark Fritz</p>
+            <p class="views-side">1K views</p>
+          </div>
+        </div>
+      </main>
+
+      <div class="profile-section">
```
Comment (edi-sipka) [inline index.html:143]: Please use a section tag instead of div.
When you are starting a new section it is important to use a section tag.

**Review by edi-sipka — COMMENTED** [review]
# Changes required ♻️

Please check my comments below and make all required changes.

Submit for re-review when you are done. 

Thanks ☺️

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @edi-sipka  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ Great Spots on the Required changes

## Required Changes ♻️
- [ ] Instead of using "You should", what do you think about using "Kindly" or "Please". Reviewees are usually much more open to receiving and acknowledging feedback when it's in a less demanding and more suggestive tone. Kindly edit your comments to a more suggestive tone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @edi-sipka  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**